loader/loader.go

```diff
@@ -968,11 +975,14 @@ func fetchMatchedLiteral(ctx *tcontext.Context, router *router.Table, schema, ta
 func (l *Loader) restoreData(ctx context.Context) error {
 	begin := time.Now()
 
-	conn, err := createConn(l.cfg)
+	baseDB, conn, err := createConn(ctx, l.cfg)
```

Comment:
the same name (`conn`) with the imported pkg name.

Reply:
change [it](https://github.com/pingcap/dm/pull/266/files#diff-b38e43f0fb016af3e8cecd2d12d779ecR984)